### ISSUE TITLE
../configure && make doesn't work in a subdirectory

### ISSUE THREAD

**Issue by sjackman** [walaj/bxtools#7]
It's helpful to be able to build in a subdirectory when working with multiple architectures, such as both Linux and macOS.

```sh
mkdir build
cd build
../configure
make
```

```
make  all-recursive
Making all in SeqLib/htslib
/bin/sh: line 0: cd: SeqLib/htslib: No such file or directory
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

**Comment by walaj** [comment]
Will look into this. Not immediately sure how to get autotools to do this.

**Comment by sjackman** [comment]
It's usually a matter of using `$srcdir`, `$builddir`,  `$top_srcdir`, and `$top_builddir` as appropriate in the `Makefile.am`.

**Comment by walaj** [comment]
Yeah, that's where I'm stuck. It works in the other ``Makefile.am`` files, but not the main one. I'm seeing this after running ``automake``:

``Makefile.am:2: error: required directory ./$(srcdir)/SeqLib/htslib does not exist``

with this ``Makefile.am``

```bash
AUTOMAKE_OPTIONS = foreign                                                                                                                                                                                
SUBDIRS = $(srcdir)/SeqLib/htslib $(srcdir)/SeqLib/src src
```

Not sure why that doesn't work...

**Comment by sjackman** [comment]
`walaj/htslib` doesn't use `automake`. There's no `Makefile.am` nor `Makefile.in` file, just a bare `Makefile`.  It may be possible to hack it up to get it working, but it may not be worth the effort.
